filament/src/details/VertexBuffer.cpp

```diff
@@ -206,7 +206,11 @@ void FVertexBuffer::terminate(FEngine& engine) {
             driver.destroyBufferObject(bo);
         }
     }
-    driver.destroyVertexBuffer(mHandle);
+    if (mBoneBufferObjectsUsed){
+        driver.destroyBufferObject(mBoJointsHandle);
+        driver.destroyBufferObject(mBoWeightsHandle);
```

Comment:
use `Bone` instead of `Bo`

Reply:
Bo means BufferObject not Bone.  I thought the names "mBufferObjectJointsHandle" and "mBufferObjectWeightHandle" were too long. I will rename them.